test/pylib/util.py

```diff
@@ -44,6 +46,10 @@ def unique_name():
     return unique_name_prefix + str(current_ms)
 
 
+def random_string(length=10, chars=string.ascii_lowercase + string.digits):
+    return ''.join(random.choice(chars) for x in range(length))
```

Comment:
`random_string` is too vague IMO, this function was clearly designed for constructing identifiers. It's an important property (not all strings can be used as parts of identifiers), so I'd change the name (e.g. `random_identifier`) and would also make sure that it's safe to use it as an identifier itself (the name can start with a digit, not sure if CQL allows that).

Maybe we could even allow to customize the prefix:

```python
random_identifier(prefix="deleted_user")
```

Reply:
Ah, actually now I see that `random_string` was taken directly from cql-pytest. There, we also have `unique_name`. How about introducing/using something like that for the sake of consistency?

Reply:
Yes I took random_string from cql, I didn't want to use unique_name because since it's time based collision across python instances is much greater (probably low, but still)

Reply:
I see (I also see that we already have `unique_name` in test.py tests).

Maybe `unique_name` should be adjusted to generate random identifiers, then? This should make other existing tests less prone to potential issues. You could keep the existing timestamp part and append some random string at the end (we're kinda reinventing timeuuids right now, but in a more readable form).

Reply:
I can add random string suffix to unique_name if that won't break anything. This is what you're proposing?

Reply:
Yes, that's what I meant.